packages/cli/lib/lib/webpack/webpack-client-config.js

```diff
@@ -84,7 +84,7 @@ function clientConfig(env) {
 
 		plugins: [
 			new PushManifestPlugin(env),
-			...RenderHTMLPlugin(env),
+			...(await RenderHTMLPlugin(env)),
```

Comment:
maybe we should lowercase this, looks like a class here

Reply:
Done